doc/ref/clouds/all/salt.cloud.clouds.rackspace.rst

```diff
@@ -1,6 +0,0 @@
-===========================
```

Comment:
Can you remove the reference to this file in `doc/ref/clouds/all/index.rst`, too?